sphinx/templates/html_theme_extension/setup.py_t

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages
+
+setup(
+    name='sphinxcontrib-themes-{{ name }}',
+    version='1.0',
+    packages=find_packages(),
+    include_package_data=True,
+    entry_points="""
+        [sphinx.html_themes]
+        {{ name }} = sphinxcontrib.themes.{{ name }}
+    """,
```

Comment:
**IMPORTANT**: If we provide "namespace package" layout, `namespace_packages` is needed.
```
namespace_packages=['sphinxcontrib', 'sphinxcontrib.themes'],
```